webapp-django/crashstats/api/tests/test_views.py

```diff
@@ -28,6 +28,9 @@
     Reprocessing,
```

Comment:
Almost all the changes in this file is for changing the mocking now that the crash_data comes from an implementation instead of the middleware (json).